src/components/ui/Button/Button.tsx

```diff
@@ -1,11 +1,39 @@
+import type { ReactNode } from 'react'
 import React from 'react'
-import { Button as UIButton } from '@faststore/ui'
+import { Button as UIButton, Icon as UIIcon } from '@faststore/ui'
 import type { ButtonProps } from '@faststore/ui'
 
-type Props = ButtonProps
+import './buttons.scss'
 
-function Button(props: Props) {
-  return <UIButton {...props} />
+type Variant = 'primary' | 'secondary'
+
+type Props = ButtonProps & {
+  variant?: Variant
+  inverse?: boolean
+  iconLeft?: ReactNode
+  iconRight?: ReactNode
+}
+
```

Comment:
Maybe we can use like this
```suggestion
type IconPosition = 'left' | 'right'

type Props = ButtonProps & {
  variant?: Variant
  inverse?: boolean
  iconPosition?: IconPosition
}

```

Reply:
Never mind! You are using other approach. :)

Reply:
Hmm, since we won't permit to use icons on each side. I agree with @hellofanny so we can pass something like:
```suggestion

type Variant = 'primary' | 'secondary'
type IconPosition = 'left' | 'right'

type Props = ButtonProps & {
  variant?: Variant
  inverse?: boolean
  icon?: ReactNode
  iconPosition?: IconPosition
}

```